### DISCUSSION TITLE
Please add ZeptoMail support

### DISCUSSION THREAD

**Discussion by naveencom** [anymail/django-anymail#404] [Ideas]
Hi,

Can you please add [ZeptoMail](https://www.zoho.com/zeptomail/pricing.html) as a supported ESP ?

ZeptoMail is part of the Zoho suite of apps, which is known for its affordability. 

ZeptoMail has a pay-as-you-go option. It's beneficial if we have seasonally high requirements (like during the holidays). The credits we purchase are valid for six months, so there's enough time to utilize them all and not pay for more emails than you're using.

**Comment by medmunds** [comment]
[To express "+1" support for this, use the (↑) upvote button at the bottom of the original message.]

ZeptoMail seems like a great fit for Anymail:

- It's for transactional email (not bulk marketing lists)
- It has an HTTP sending API (four actually: single, batch, single-template, batch-template)
- It has free send allowance that would let us develop and test the Anymail integration[^1]
- It supports tracking webhooks (for bounce, open, and click tracking only)

From what I can tell, ZeptoMail does not support a webhook for inbound email. (Possibly that's some other product in the Zoho suite.)

I'm unlikely to get to this before the end of the year, but if anyone else wants to start an integration before then a PR would be welcome.

[^1]: The free allotment is 10,000 messages, which would cover a decade or more of regular Anymail integration tests. But I can't tell if there's a time limit on it. If so, we'd either need Zoho to create a free testing account for Anymail's ongoing low-volume testing, or we'd have to treat ZeptoMail as an "unsupported" integration after the free trial period.

**Reply by naveencom** [reply]
Thanks for the consideration.

**Reply by naveencom** [reply]
@medmunds Zoho has released a Django integration: https://www.zoho.com/zeptomail/django-integration.html

**Reply by medmunds** [reply]
> Zoho has released a Django integration

Thanks for the pointer. If Zoho's integration meets your needs, there's no need to involve django-anymail in the mix. (We may still want to add a ZeptoMail integration to Anymail in the future, to simplify migrating from other ESPs to ZeptoMail and vice versa.)

Also, just a standard reminder that Django works with *any* email service provider that supports SMTP (virtually all do, [including ZeptoMail](https://www.zoho.com/zeptomail/help/smtp-home.html)) by using Django's built-in [SMTP EmailBackend](https://docs.djangoproject.com/en/stable/topics/email/#smtp-backend) with the appropriate EMAIL_HOST and other EMAIL_ settings.
